lib/Auth/Process/COmanageDbClient.php

```diff
@@ -0,0 +1,480 @@
+<?php
+
+/**
+ * COmanage authproc filter.
+ *
+ * This class is the authproc filter to get attributes from rest api
+ *
+ * Example configuration in the config/config.php
+ *
+ *    authproc.aa = array(
+ *       ...
+ *       '60' => array(
+ *            'class' => 'attrauthcomanage:COmanageDbClient',
+ *            'coId' => 2,
+ *            'userIdAttribute' => 'eduPersonUniqueId',
+ *       ),
+ *
+ * @author Nicolas Liampotis <nliam@grnet.gr>
```

Comment:
Update author list

Reply:
Changes made in "Update COmanageDbClient description and authors" commit